analyze/csv.rs

```diff
@@ -1,7 +1,7 @@
-use ir::Id;
-
 #[derive(Debug, Default, Serialize)]
 pub struct CsvRecord {
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Is every command using the same csv records? I find this mildly surprising, and I assumed that they would each have their own specific csv record type, rather than adding options to all the fields and skipping `None`s. That would be a bit less "dynamically typed" although it is more boilerplate.

Did you consider that approach? Why did you end up going down this route?

Reply:
I created a "common" `CsvRecord` because most part of the records shared the fields from `id` to `size_related_stuff`, and the other commands like `dominator` might have one or two extra fields.
Declaring new `Record` inside every method seems a bit trivial, but duplicated skipping `None`s too.
It seems you prefer the individual record, I'll fix that.